### PR TITLE
🧹 It's easy to manage and neat. Saves it in '/pydoro.ini'  to '/.config/pydoro/pydoro.ini.'

### PR DESCRIPTION
**Pull request by sachinsenal0x64**
Cleaning your Home! :)

### CODE REVIEW

File: pydoro/pydoro_core/config.py [hunk 1]
```diff
@@ -95,10 +95,16 @@ def _ini_parse(self):
...
         """
-        with open(filename, "w+") as configfile:
-            self._conf.write(configfile)
+        try:
+            with open(filename, "w+") as configfile:
+                self._conf.write(configfile)
+                
+        except FileNotFoundError:
+            config_dir = os.path.expanduser("~/.config/pydoro")
+            os.makedirs(config_dir, exist_ok=True)
```
Comment (JaDogg) [inline pydoro/pydoro_core/config.py]: Can we move this before attempting to write the file?

File: pydoro/pydoro_core/config.py [hunk 2]
```diff
@@ -95,10 +95,16 @@ def _ini_parse(self):
...
+            os.chdir(config_dir)
```
Comment (JaDogg) [inline pydoro/pydoro_core/config.py]: Why is this required?

File: pydoro/pydoro_core/config.py [hunk 3]
```diff
@@ -95,10 +95,16 @@ def _ini_parse(self):
...
+        Saves it in '~/.config/pydoro/pydoro.ini' or the location specified by PYDORO_CONFIG_FILE environment variable
         """
-        with open(filename, "w+") as configfile:
-            self._conf.write(configfile)
+        try:
+            with open(filename, "w+") as configfile:
+                self._conf.write(configfile)
+                
+        except FileNotFoundError:
+            config_dir = os.path.expanduser("~/.config/pydoro")
```
Comment (JaDogg) [inline pydoro/pydoro_core/config.py]: Move this line prior to the with open as well.

**Review by JaDogg — CHANGES_REQUESTED** [review]

File: pydoro/pydoro_core/config.py (hunk 1 above)
Comment (sachinsenal0x64) [reply pydoro/pydoro_core/config.py]: Done :)

**Review by JaDogg — APPROVED** [review]

**Comment by sachinsenal0x64** [comment]
Cool
